build/WebRICE_styles.css

```diff
@@ -106,16 +135,14 @@
   display: var(--module-visibility);
   flex-direction: column;
   margin: 0.15rem 0 0 0;
-  position: absolute;
+  position: fixed;
   background-color: var(--main-color-two);
-  max-width: 13rem;
 }
 
 #settingsContainer {
   overflow-y: scroll;
```

Comment:
I know this isn't introduced in this comment in this PR, but shouldn't this be auto, so that the scroll bar (white margin in my case) isn't visible when there's no overflow to scroll?

Reply:
Good point. It's not made in this PR but it's related to improving the look of the settings module so makes sense to change it now.